x-pack/test/api_integration/apis/uptime/graphql/monitor_states.ts

```diff
@@ -154,6 +167,22 @@ export default function({ getService }: FtrProviderContext) {
           expect(nonSummaryRes.monitorStates.summaries.length).to.eql(0);
         });
 
+        it('should return a monitor with mix state if check status filter is down', async () => {
+          const params = makeApiParams(testMonitorId);
+          params.statusFilter = 'down';
+
+          const nonSummaryRes = await getMonitorStates(params);
+          expect(nonSummaryRes.monitorStates.summaries.length).to.eql(1);
+        });
+
+        it('should not return a monitor with mix state if check status filter is up', async () => {
```

Comment:
Since we're now testing the filter, I think we need to still test that the `up` filter returns results.

I think our test matrix should have three monitors each with two geo locations:
- One in a down state
- One in an up state
- One in a mix state

Then have three tests:
- With no filters all are returned
- With down, the mixed down and all down ones are returned
- With up only the fully up one is returned

Since this is complex code I think we'll be glad that we have this extra level of confidence in the future.